### PR TITLE
Refactor SplitToken&Supersede

### PR DESCRIPTION
**Pull request by PKUzjw**

### CODE REVIEW

File: src/components/home/elements/SplitTokenOrSupersede.vue [hunk 1]
```diff
@@ -538,10 +679,11 @@ export default {
             this.paused = true
             try {
                 let jsonObj = JSON.parse(decodeString.replace(/"amount":(\d+)/g, '"amount":"$1"')) // The protocol defined amount must use Long type. However, there is no Long type in JS. So we use BigNumber instead. Add quotes (") to amount field to ensure BigNumber parses amount without precision loss.
-                this.newIssuer = jsonObj.address
+                var newIssuer = jsonObj.address
```
Comment (SheldonYS) [inline src/components/home/elements/SplitTokenOrSupersede.vue]: why need define `newIssuer` here? `this.newIssuer` is enough to handle issuer.

File: src/components/home/elements/SplitTokenOrSupersede.vue [hunk 2]
```diff
@@ -551,19 +693,21 @@ export default {
...
                     this.qrErrMsg = 'Invalid address of new issuer.'
                 } else {
                     this.qrErrMsg = void 0
                 }
             } catch (e) {
-                if (this.isValidIssuer(decodeString)) {
-                    this.newIssuer = decodeString
+                this.newIssuer = decodeString
+                if (this.isValidIssuer) {
+                    newIssuer = decodeString
```
Comment (SheldonYS) [inline src/components/home/elements/SplitTokenOrSupersede.vue]: The same here. The code can be simplify with use `this.newIssuer` only. No need local variable  `newIssuer`.

File: src/components/home/elements/SplitTokenOrSupersede.vue [hunk 3]
```diff
@@ -465,20 +599,20 @@ export default {
...
-            if (this.pageId === 1) {
-                this.$refs.sendModal.hide()
-            } else {
-                this.pageId--
+            let pageId = this.walletType === 'hotWallet' ? --this.pageId : --this.coldPageId
+            if (pageId === 0) {
+                this.$refs.splitTokenOrSupersedeModal.hide()
             }
         },
         coldPrevPage() {
```
Comment (Haynar216) [inline src/components/home/elements/SplitTokenOrSupersede.vue]: delete this method

File: src/components/home/elements/SplitTokenOrSupersede.vue [hunk 4]
```diff
@@ -490,19 +624,26 @@ export default {
...
-            this.newIssuer = ''
             this.pageId = 1
             this.coldPageId = 1
-            this.scanShow = false
-            this.qrInit = false
-            this.paused = false
-            this.qrErrMsg = void 0
             this.sendError = false
             this.coldSignature = ''
+            if (this.functionName === 'Split Token') {
```
Comment (Haynar216) [inline src/components/home/elements/SplitTokenOrSupersede.vue]: All data should be reset

File: src/components/home/elements/SplitTokenOrSupersede.vue [hunk 5]
```diff
@@ -20,25 +20,38 @@
...
           v-if="pageId===1">
-          <b-form-group label="Maker Wallet Address"
+          <b-form-group :label="functionName === 'Split Token' ? 'Issuer Wallet Address' : 'Maker Wallet Address'"
                         label-for="address-input">
-            <b-form-input id=address-input
+            <b-form-input v-if="functionName === 'Split Token'"
+                          id=address-input
+                          class="address-input"
+                          readonly
+                          v-model="issuer"
```
Comment (Haynar216) [inline src/components/home/elements/SplitTokenOrSupersede.vue]: Try to use new variable 'issuerOrMaker' to simplify code

File: src/components/home/elements/SplitTokenOrSupersede.vue [hunk 6]
```diff
@@ -176,19 +237,59 @@
...
+            <b-form-invalid-feedback id="inputLiveFeedback"
+                                     v-else-if="isGreaterThanMax">
+              Unity is too large. Please change it to an integer less than 9223372036854775808.
+            </b-form-invalid-feedback>
+            <b-form-invalid-feedback id="inputLiveFeedback"
+                                     v-else>
+              Invalid Unity.
+            </b-form-invalid-feedback>
+          </b-form-group>
+          <b-form-group v-if="functionName === 'Split Token'">
```
Comment (Haynar216) [inline src/components/home/elements/SplitTokenOrSupersede.vue]: Supersede also needs to show transaction fee.

**Pull request merged**